### PR TITLE
Update README.md to clarify version support and add headers where applicable

### PR DESCRIPTION
**Pull request by BrailleBennett**
.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,10 +3,10 @@ Various workarounds for Geyser players that modify the server in order to achiev
...
 - Bamboo and dripstone collision (by setting them to no server-side collision)
-- https://github.com/GeyserMC/Geyser/issues/638 by implementing the control server-side. (Doesn't work on 1.20+)
+- Allow Geyser players to ride pigs (by implementing the control server-side.) (Doesn't work on 1.20+)
 
-Supported Versions:
-- 1.14.x - 1.19.x
-- 1.17.x & above for pig riding
+## Supported Versions:
+- 1.14.x - 1.20.x for bamboo and dripstone fix
+- 1.17.x - 1.19.x for pig riding
```
Comment (onebeastchris) [inline README.md:12]:
```suggestion
- 1.17.x - 1.19.x for pig/strider riding
```

File: README.md [hunk 2]
```diff
@@ -3,10 +3,10 @@ Various workarounds for Geyser players that modify the server in order to achiev
...
+## Fixes:
 - Bamboo and dripstone collision (by setting them to no server-side collision)
-- https://github.com/GeyserMC/Geyser/issues/638 by implementing the control server-side. (Doesn't work on 1.20+)
+- Allow Geyser players to ride pigs (by implementing the control server-side.) (Doesn't work on 1.20+)
 
-Supported Versions:
-- 1.14.x - 1.19.x
-- 1.17.x & above for pig riding
+## Supported Versions:
+- 1.14.x - 1.20.x for bamboo and dripstone fix
```
Comment (onebeastchris) [inline README.md:11]:
```suggestion
- 1.14.x - 1.20.x for bamboo and dripstone collision fixes 
```

File: README.md [hunk 3]
```diff
@@ -3,10 +3,10 @@ Various workarounds for Geyser players that modify the server in order to achiev
 
 Issues with each workaround are listed in the plugin's config. **Please take your time to read them as the workarounds in this plugin can be used for exploitative purposes.**
 
-Fixes:
+## Fixes:
 - Bamboo and dripstone collision (by setting them to no server-side collision)
-- https://github.com/GeyserMC/Geyser/issues/638 by implementing the control server-side. (Doesn't work on 1.20+)
```
Comment (onebeastchris) [inline README.md:8]: Why was this removed? That's not yet resolved - unless you intend to wait with this PR until that issue was resolved? For the latter, I'd turn this PR into a draft until then.

**Review by onebeastchris — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (BrailleBennett) [reply README.md:8]: I removed the link to the issue and described what the fix did in words to make it easier to understand for the average user what exactly was being fixed. Should I link the resolved issue as well as having the human readable text?

Comment (onebeastchris) [reply README.md:8]: Yep, that'd be ideal.

**Comment by BrailleBennett** [comment]
Can we check if bamboo and drip stone is functional in 1.21? If so, I can update the line that refrences the supported versions.

**Comment by onebeastchris** [comment]
Bamboo/Dripstone is still functional from my brief testing

File: README.md [hunk 4]
```diff
@@ -3,10 +3,11 @@ Various workarounds for Geyser players that modify the server in order to achiev
...
+- Allow Geyser players to ride pigs and striders (by implementing the control server-side.) (Doesn't work on 1.20+)
+	Fixes https://github.com/GeyserMC/Geyser/issues/638
 
-Supported Versions:
-- 1.14.x - 1.19.x
-- 1.17.x & above for pig riding
+## Supported Versions:
+- 1.14.x - 1.20.x for bamboo and dripstone collision fixes
```
Comment (onebeastchris) [inline README.md:12]:
```suggestion
- 1.14.x - 1.21 for bamboo and dripstone collision fixes 
```

**Review by onebeastchris — APPROVED** [review]

**Pull request merged**